WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -124,25 +128,27 @@ public class CurrencyFormatter {
     /// Applies currency option settings to the amount (as String) for the given currency.
     ///
     /// - Parameters:
-    ///     - amount: a raw string representation of the amount, from the API, with no formatting applied. e.g. "19.87"
+    ///     - stringAmount: a raw string representation of the amount, from the API, with no formatting applied. e.g. "19.87"
```

Comment:
Maybe we can rename the parameter `stringAmount` to `amount`? It seems clear from the type in the method that it's a String amount.